warp10/src/main/java/io/warp10/continuum/egress/Egress.java

```diff
@@ -49,6 +49,7 @@
   private static final String[] REQUIRED_PROPERTIES = new String[] {
     Configuration.EGRESS_HOST,
     Configuration.EGRESS_PORT,
+    Configuration.EGRESS_TCP_BACKLOG,
```

Comment:
since there is a default value assigned, this should not be a required property.